ci/test.sh

```diff
@@ -47,6 +47,7 @@ mvn -B verify --file pom.xml \
 -P \${C3}it-cassandra-3.11 \
 -P \${C4}it-cassandra-4.0 \
 -P \${DSE}dse -P \${DSE}it-dse-6.8 \
+-P grpc \
```

Comment:
maybe it's easier to do `-P default,grpc` instead of adding all the modules to `grpc`?

Reply:
The `default` profile conflicts with the `dse` profile now, AFAIK.

Reply:
We could probably achieve what @mpenick  is suggesting by adding another flag like `${DSE}` and then `-P \${DEFAULT}default`

Reply:
I've tried `-P default,grpc` but didn't work

Reply:
I use `-P default,dese` all the time :)

Reply:
I tested it again, and the `-P default,dse` seems to work - the application is compiling. 
However, for the `-P dse,grpc` it does not and fails with:
```
[INFO] io.stargate:stargate ............................... SUCCESS [  0.562 s]
[INFO] core ............................................... SUCCESS [  4.031 s]
[INFO] stargate-starter ................................... SUCCESS [  5.042 s]
[INFO] persistence-api .................................... SUCCESS [  9.911 s]
[INFO] persistence-common ................................. SUCCESS [  0.493 s]
[INFO] persistence-test ................................... SUCCESS [  0.227 s]
[INFO] authnz ............................................. SUCCESS [  0.524 s]
[INFO] persistence-cassandra-3.11 ......................... SUCCESS [ 10.102 s]
[INFO] persistence-cassandra-4.0 .......................... SUCCESS [ 11.511 s]
[INFO] persistence-dse-6.8 ................................ FAILURE [ 19.276 s]
[INFO] cql ................................................ SKIPPED
[INFO] metrics-jersey ..................................... SKIPPED
[INFO] graphqlapi ......................................... SKIPPED
[INFO] restapi ............................................ SKIPPED
[INFO] auth-api ........................................... SKIPPED
[INFO] auth-table-based-service ........................... SKIPPED
[INFO] auth-jwt-service ................................... SKIPPED
[INFO] health-checker ..................................... SKIPPED
[INFO] config-store-api ................................... SKIPPED
[INFO] config-store-yaml .................................. SKIPPED
[INFO] rate-limiting-global ............................... SKIPPED
[INFO] grpc-proto ......................................... SKIPPED
[INFO] testing-services ................................... SKIPPED
[INFO] testing ............................................ SKIPPED
[INFO] grpc ............................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:03 min
[INFO] Finished at: 2021-07-26T12:02:35+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project persistence-dse-6.8: Could not resolve dependencies for project io.stargate.db.dse:persistence-dse-6.8:jar:1.0.31-SNAPSHOT: Failed to collect dependencies at com.datastax.dse:dse-core:jar:6.8.13: Failed to read artifact descriptor for com.datastax.dse:dse-core:jar:6.8.13: Could not transfer artifact com.datastax.dse:dse-core:pom:6.8.13 from/to datastax-artifactory (https://repo.sjc.dsinternal.org/artifactory/dse): Transfer failed for https://repo.sjc.dsinternal.org/artifactory/dse/com/datastax/dse/dse-core/6.8.13/dse-core-6.8.13.pom: Unknown host repo.sjc.dsinternal.org: nodename nor servname provided, or not known -> [Help 1]
[ERROR] 
```